portfolio/src/App.css

```diff
@@ -1,5 +1,12 @@
+/* https://coolors.co/05668d-427aa1-ebf2fa-679436-a5be00 - Color Palette */
+/* 131112 - Smoky Black, FFE74C - Minion Yellow, 7DC95E - Mantis
+EDAE49 - Sunray, F9C80E - Jonquil
+*/
+@import url('https://fonts.googleapis.com/css?family=Archivo|Questrial|Baloo+2');
+
 .App {
```

Comment:
Not sure I like "App"

And really it should also be using BEM at least for sub components. Google it :)

Reply:
Don't get what you mean by this comment. Would you prefer for this to be renamed basically?

Reply:
Yeah, rename it I think